08-rails/better-errors-and-state.md

```diff
@@ -1,12 +1,12 @@
-# Rails Debugging
+# Rails, State and Better Errors
```

Comment:
This is the gooooofiest suggestion, but since the file is named `better-errors-and-state`, and this headline is `State and Better Errors`, could we conform one to the other?

(I don't particularly care the order, if you decide to preserve the headline or the file name)

Reply:
I will probably conform the file name, as what we are doing is debugging.

Reply:
wait, misread, other way around